stdlib/ballerina-http/src/main/ballerina/http/http_connection.bal

```diff
@@ -55,10 +55,11 @@ public type Connection object {
     documentation {
         Cancels the handshake.
 
-        P{{status}} Status code for closing the connection
-        P{{reason}} Reason for closing the connection
+        P{{status}} Error Status code for cancelling the upgrade and closing the connection
+        P{{reason}} Reason for cancelling the upgrade
+        R{{}} An `error` in the case of cancelling the upgrade
```

Comment:
Shall we change the description of the return type to say that we are returning either error or nil?